From 980230645c8ea3b531b85401de5e4bca0f860e42 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Wojciech=20Regu=C5=82a?= <wojciechregula@yahoo.pl>
Date: Tue, 20 Feb 2018 20:17:48 +0100
Subject: [PATCH] Fixing security issue

Plugin logs sensitive information and that is considered as a security issue. iOS logs are can be easily accessed by attacker.

References:
https://www.owasp.org/index.php/Mobile_Top_10_2016-M2-Insecure_Data_Storage
https://www.kaspersky.com/blog/starbucks-moves-quick-to-fix-application-security-vulnerability/3510/ (Starbucks for instance had problem with placing sensitive data to iOS logs and that was a big scandal ;-) )
---
 src/ios/CDVKeychain.m | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/ios/CDVKeychain.m b/src/ios/CDVKeychain.m
index dbc0a40..6c2fea4 100644
--- a/src/ios/CDVKeychain.m
+++ b/src/ios/CDVKeychain.m
@@ -69,9 +69,7 @@ - (void) set:(CDVInvokedUrlCommand*)command {
     NSString* key = [arguments objectAtIndex:0];
     NSString* value = [arguments objectAtIndex:1];
     BOOL useTouchID = [[arguments objectAtIndex:2] boolValue];
-
-      NSLog(@"SET %@ %@ %d", key ,value, useTouchID);
-
+   
     A0SimpleKeychain *keychain = [A0SimpleKeychain keychain];
 
     if(useTouchID) {
